tests/local/Earthfile

```diff
@@ -32,6 +32,30 @@ test-local:
         fi
 
 
+##################################################
+# tests for running locally with --allow-parallel
+
+test-parallel:
```

Comment:
This isn't really testing parallelization - perhaps add a comment that it's merely a smoke test, and not a test verifying that the targets indeed execute in parallel.